frontend/src/components/algolia/algoliaInstantSearch.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import algoliasearch from 'algoliasearch/lite';
+import {
+  Highlight, Hits, InstantSearch,
+
+
+  Pagination, SearchBox
+} from 'react-instantsearch-dom';
```

Comment:
Why vertical space inside import? There is prettier configured for the whole project in the root folder. If you use VSCode it should format on save…